modules/management/kubectl-apply/main.tf

```diff
@@ -41,8 +41,9 @@ data "google_container_cluster" "gke_cluster" {
 data "google_client_config" "default" {}
 
 module "kubectl_apply_manifests" {
-  for_each = local.apply_manifests_map
-  source   = "./kubectl"
+  for_each   = local.apply_manifests_map
+  source     = "./kubectl"
+  depends_on = [var.gke_cluster_exists]
```

Comment:
Where is this var defined? Is it the output of the gke-cluster module?

Reply:
Yes. Defined in `variables.tf` and passed from gke-cluster module

Reply:
That PR was already merged a while back to fix a different bug